sensing/pointcloud_preprocessor/src/outlier_filter/dual_return_outlier_filter_nodelet.cpp

```diff
@@ -231,10 +231,10 @@ void DualReturnOutlierFilterComponent::filter(
       if (deleted_azimuths.size() == 0) {
         continue;
       }
-      while ((uint)deleted_azimuths[current_deleted_index] <
+      while (current_deleted_index < (deleted_azimuths.size() - 1) &&
```

Comment:
Isn't it `current_deleted_index < deleted_azimuths.size() `

Reply:
> Please let the person who made the comment resolve the conversation (in addition to being good manners help keep track of what has been handled / answered in the PR)

OK, thank you for your advice :+1: